session/src/main/java/org/apache/iotdb/session/SessionConnection.java

```diff
@@ -67,6 +67,9 @@
   private ZoneId zoneId;
   private EndPoint endPoint;
 
+  //TestOnly
+  public SessionConnection(){}
```

Comment:
It would be better to use an annotation, and please format the code.

Reply:
Testonly's anonation is currently in the server module. The session module does not currently have a dependency on the server module, so I don't think it would be appropriate to introduce a dependency just for this purpose. As for creating a new testonly anotation in the session module, I think it's a bit redundant because it's the only one we've used so far. But if you think it's worth creating a new anonation in the Session Module, I'll do it. What your opinion?

Reply:
That is the case, we can extract that into another module later.